### ISSUE TITLE
Sortable MultiSelect: poor support for mobile accessibility in Android

### ISSUE THREAD

**Issue by jphawk** [JedWatson/react-select#5573]
There are mobile screen reader accessibility problems with the Sortable MultiSelect case. 

Tested on Huawei P30 Pro, Chrome browser (latest, 110.0.5481.153), Android 12.0.0 with TalkBack.
Most of the times only one selection from the list of options is possible with double tap. The first item in the list is always unselectable for some reason. It’s very difficult to enter the menu to select other items. Also all of the buttons for removing options can't be accessed properly with TalkBack. 

There are some minor problems with iOS's VoiceOver as well, but at least it's possible to use the component, but with Android's TalkBack it, unfortunately, doesn't work.
